stdlib/ftplib.pyi

```diff
@@ -86,7 +86,7 @@ class FTP:
     def makeport(self) -> socket: ...
     def makepasv(self) -> tuple[str, int]: ...
     def login(self, user: str = "", passwd: str = "", acct: str = "") -> str: ...
-    # In practice, `rest` rest can actually be anything whose str() is an integer sequence, so to make it simple we allow integers.
+    # In practice, `rest` rest can actually be anything whose str() is an integer sequence, so to make it simple we allow integers
```

Comment:
```suggestion
    # In practice, `rest` can actually be anything whose str() is an integer sequence, so to make it simple we allow integers
```

Reply:
y'know, I wasn't sure if this meant "the rest of `rest`", but if you agree to remove it, let's do so

Reply:
Yeah, I _think_ it's just a typo? But IDK, I'm not an `ftplib` expert 😄